mappings/net/minecraft/advancement/criterion/ItemDurabilityChangedCriterion.mapping

```diff
@@ -10,8 +10,11 @@ CLASS net/minecraft/class_2069 net/minecraft/advancement/criterion/ItemDurabilit
 		FIELD field_9639 delta Lnet/minecraft/class_2096$class_2100;
 		METHOD <init> (Lnet/minecraft/class_2073;Lnet/minecraft/class_2096$class_2100;Lnet/minecraft/class_2096$class_2100;)V
 			ARG 1 item
+			ARG 2 durability
+			ARG 3 delta
 		METHOD method_8967 create (Lnet/minecraft/class_2073;Lnet/minecraft/class_2096$class_2100;)Lnet/minecraft/class_2069$class_2071;
 			ARG 0 item
+			ARG 1 itemMinMax
```

Comment:
Is this the durability or delta range? should be `durability` or `delta`

Reply:
it is the durability

Reply:
```suggestion
			ARG 1 durability
```

Reply:
https://github.com/FabricMC/yarn/pull/1218/commits/7395d7e5537faff78f82191b3f5349be57c71d63